### PR TITLE
Supply an app context in storybook plugin test

### PR DESCRIPTION
**Pull request by freben**
Unbreaks the Plugin storybook views.

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4678-qp6jlw88ruhuodbl0oxcqdawxskzish0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60364a10f285af60af96e895

**Review by Rugvip — APPROVED** [review]

File: .changeset/six-crabs-pump.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': patch
```
Comment (Rugvip) [inline .changeset/six-crabs-pump.md:2]: Wait this is totally not needed tho

Comment (freben) [reply .changeset/six-crabs-pump.md:2]: Ah foo, of course

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4678-qp6jlw88ruhuodbl0oxcqdawxskzish0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60364a10f285af60af96e895

**Pull request merged**